__tests__/helpers/getReadingTime.test.js

```diff
@@ -0,0 +1,11 @@
+import { getReadingTime } from '../../helpers';
+import { createArticle } from '../mocks/db.json';
+
+describe('getReadingTime', () => {
+  test('Should return reading time', () => {
+    expect.assertions(1);
+    expect.stringContaining('min');
+    const readingTime = getReadingTime(createArticle.body);
```

Comment:
@filschristian What happens if you provide undefined as the argument to getReadingTime(undefined)? When testing a feature, it is advisable to have at least two case scenarios (success and fail).

Reply:
I think this scenario is not necessary because all the routes that use this helper have validation. we will not have the undefined scenario.

Reply:
> I think this scenario is not necessary because all the routes that use this helper have validation. we will not have the undefined scenario.

@oesukam

Reply:
@filschristian Unit tests don't only cover what you are currently working on, it might happen that you pass an article as the argument with an undefined body. getReadingTime() is a single unit that does not depend on what is being tested in different routes. You can only depend on routes validation in case you are doing an integration test but for your case, you are doing unit test

Reply:
@oesukam Yes, I understand. Feedback implemented, you can check. Thanks!!!!!!